src/query/tsdb/remote/codecs_test.go

```diff
@@ -206,6 +209,9 @@ func TestEncodeDecodeFetchQuery(t *testing.T) {
 		MetricsType:   storage.AggregatedMetricsType,
 		StoragePolicy: policy.MustParseStoragePolicy("1m:14d"),
 	}
+	lookback := time.Minute
+	fetchOpts.LookbackDuration = &lookback
```

Comment:
Any worth in adding one without a specified lookback to ensure it uses the config one correctly?